test/API/tattr.c

```diff
@@ -1723,164 +1744,168 @@ test_attr_dtype_shared(hid_t fapl)
     h5_stat_size_t filesize;             /* Size of file after modifications */
 #endif
     herr_t ret; /* Generic return value        */
-#else
-    (void)fapl;
-#endif
+                //#else
+    if (!(vol_cap_flags_g & H5VL_CAP_FLAG_STORED_DATATYPES)) {
+        (void)fapl;
```

Comment:
You should be able to remove this block since at least one usage of `fapl` will be compiled in now. This was just to silence a compiler warning about `fapl` not being used in the code.

Reply:
Okay, I will remove this block.